blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,275 @@
+Some also prefer to go old school and use [Vim](https://www.vim.org/), [Emacs](https://www.gnu.org/software/emacs/), or the newer fork of Vim, [Neovim](https://neovim.io/)
+(which actually has all the language support that VSCode has if you are willing to configure it yourself). While your choise of editor doesn't impact the quality of code
+which you will produce, it is crucial that you choose one and become comfortable working in it as much of your productivity will come from your ability to use your editor.
+
+### Running Knative Locally
+
+Now that you have verified that you have all the prerequisites, let's get started setting up your local system to run Knative!
+
+#### Docker/Podman Setup
+
+Knative runs in a Kubernetes Cluster, which means it runs as a set of Containers. You can think of a container as something which both contains your code and all the
+dependencies needed to run your code, as well as something which runs your code with those packaged dependencies. The two most common tools used to build and run containers
+are __Docker__ and __Podman__, so we will discuss setting both up here. However, do note that we think that __using Docker__ is easier, and as such, we recommend that you use Docker.
+
+##### Option 1: Setting up Docker
+
+The first step to setting up Docker is to install it. The easiest way to do this is to [install Docker Desktop](https://docs.docker.com/get-docker/), although you only need
+the Docker Engine for Knative so you can also just [install the Docker Engine](https://docs.docker.com/engine/install/). Note: if you plan on just installing the Docker Engine,
+make sure that you complete the post-install steps.
+
+Once you have Docker installed on your system, you will need to log in to a container registry. There are multiple different registries which you can use.
+Firstly, you can use a Docker account to sign into Docker hub. To create one, you can [sign up for one here](https://hub.docker.com/signup). Once you have an account, 
+all you need to do is type `docker login`, and follow the instructions. Another option is to use a registry on [Quay.io](https://quay.io/), or to 
+[create a local registry](https://www.allisonthackston.com/articles/local-docker-registry.html). If you use either of these approaches, or some other registry,
+all you need to do is login with `docker login -u <username> -p <password> <registry_url>`. You only need to include the `-u <username` and `-p <password` options
+if the registry needs you to sign in. For example, to login to a local registry, all you need to do is run `docker login localhost:8080`, assuming that the registry
+is running on `localhost:8080` and you have not enabled authentication on it.
+
+##### Option 2: Setting up Podman
+
+If you want to use Podman instead of Docker, you're in luck - they have compatible CLIs! So, all you have to do is [install Podman](https://podman.io/docs/installation)
```

Comment:
```suggestion
If you want to use Podman instead of Docker, you're in luck - they have compatible CLIs! So, all you have to do is [install Podman](https://podman.io/docs/installation){:target="_blank"}
```